pkgs/os-specific/linux/nftables/default.nix

```diff
@@ -35,6 +35,13 @@ stdenv.mkDerivation rec {
       python3.pkgs.setuptools
     ];
 
+  patches = [ ./fix-py-libnftables.patch ];
```

Comment:
this could be 0 rebuilds if we only apply this, when `withPython` is `true`. not sure if that's worth the effort. in general it makes sense to have these patches unconditional, since they bitrot less easily that way.

Reply:
I think it's better to have this unconditional to prevent bitrot.